doc/man1/openssl-enc.pod

```diff
@@ -421,6 +426,10 @@ certain parameters. So if, for example, you want to use RC2 with a
 
 The default digest was changed from MD5 to SHA256 in OpenSSL 1.1.0.
 
+The B<-list> option was added in OpenSSL 1.1.1e.
```

Comment:
This assumes the PR will be merged also to the 1.1.1 branch which probably won't happen because it is not a bugfix.

Reply:
Thank you.  Is there a more appropriate branch to identify where PR will be merged?

Reply:
@t8m actually my plan was to backport this (but without the 3.0 deprecation markers) to 1.1.1, unless the @openssl/omc disagrees with my argumentation.The fact that the `openssl enc` command has a listing option (awkwardly called `-ciphers` even though it's internally named `OPT_LIST`) and the `openssl dgst` has not (although the implementation is mostly anologous), is one of the uncountable inconsistencies of the openssl commandline api IMHO and fixing that would be worth backporting. It's not a bug fix in the strict sense, but it fixes a technical debt.

_Note: Unfortunately, during the last rebase and cleanup I squashed too much. I should have left the changes which add the deprecation markers as a fixup, so it can easily be omitted while backporting. I will restore the fixup some time today, this will save me an extra pull request if I get the approval for backporting._

Reply:
See also https://github.com/openssl/openssl/pull/9912#issuecomment-534504953.